pkg/api/types.go

```diff
@@ -669,6 +671,11 @@ type V20180331ARMManagedContainerService struct {
 	*v20180331.ManagedCluster
 }
 
+// CustomCloudProfile Represents custom cloud profile
+type CustomCloudProfile struct {
+	Enviornment *azure.Environment `json:"environment,omitempty"`
```

Comment:
typo: `Enviornment` should be `Environment` (you'll want to do a global search/replace on all files for this)